Elemental/Components/AeTable/AePaginatedTableMerged.razor

```diff
@@ -0,0 +1,125 @@
+﻿@namespace Elemental.Components
+@typeparam T
+@inherits AeBaseTable<T>
+@implements IPageableComponent
+
+<div class="@_className" @attributes="InputAttributesWithoutClass">
+    @if (Headers != null && Headers.Any())
+    {
+        <AeTableRow Header GridStyle="@_gridStyle">
+            @foreach (var (header, index) in Headers.Select((h, index) => (h, index)))
+            {
+                @if (@CustomHeader != null)
+                {
+                    @CustomHeader(header, index)
+                }
+                else
+                {
+                    @DefaultHeader(header, index)
+                }
+            }
+        </AeTableRow>
+    }
+
+    <div class="ae table-rows">
+        @for (var itemIndex = 0; itemIndex < PageItems.Count; itemIndex++)
+        {
+            var item = getPageItem(itemIndex);
+            var currentIndex = itemIndex;
+            var idx = itemIndex + Page * PageSize;
+            <AeTableRow @key=idx GridStyle="@_gridStyle"
+                    tabindex="0"
+                    @onclick="(() => OnRowClick.InvokeAsync(getPageItem(currentIndex)))"
+                    @onfocus="(() => OnRowFocus.InvokeAsync(getPageItem(currentIndex)))">
+                @foreach (var (accessor, index) in Accessors.Select((a, i) => (a, i)))
+                {
+                    if (Renderers?[index] is null)
+                    {
+                        <AeTableCell>
+                            @accessor?.Invoke(item)
+                        </AeTableCell>
+                    }
+                    else
+                    {
+                        <AeTableCell>
+                            @Renderers[index](item)
+                        </AeTableCell>
+                    }
+                }
+            </AeTableRow>
+
+        }
+</div>
+</div>
```

Comment:
Could this not just be an `AePaginatedTable` component?

Reply:
You are absolutely correct, I've modified that and pushed the changes!